include/quicly/cc.h

```diff
@@ -40,6 +40,11 @@ extern "C" {
 #define QUICLY_MIN_CWND 2
 #define QUICLY_RENO_BETA 0.7
 
+#define QUICLY_SEARCH_DELV_BIN_COUNT 10                 // number of search delivered bytes bins
```

Comment:
should also have parenthesis to match other #defines

Reply:
I agree.